### ISSUE TITLE
[SF_ACC][ui5-list][ui5-li]: no tooltip provided when list item text was truncated

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#7372]
### Bug Description

No tooltip provided when list item text was truncated.
Please see https://wiki.one.int.sap/wiki/display/Acc/Truncation under "Methods to get Access to Non-Truncated Content". 
At least one alternative way needs to be provided to see the full untruncated text to satisfy Accessibility.

### Affected Component

ui5-list

### Expected Behaviour

Tooltip will be provided when list item text was truncated.

### Steps to Reproduce

1. Open page https://codesandbox.io/s/ui5-webcomponents-forked-v4flyv?file=/index.html&resolutionWidth=320&resolutionHeight=675
2. List item text was truncated, but no tooltip was provided, you can not know the full text.

### Log Output, Stack Trace or Screenshots

<img width="708" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/13098801/9f11d0e2-cdf7-44b8-9eb1-058196f72168">

### Priority

Medium

### UI5 Web Components Version

1.15.x

### Browser

Chrome, Edge

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @ui5-webcomponents-topic-p I am forwarding this issue to you, because there is no tooltip provided for ui5-li, when text is truncated.

Regards,
Iliana

**Comment by yanaminkova** [comment]
Internal Reference: BGSOFUIPIRIN-6187

**Comment by Harshithagc18** [comment]
Any progress on this feature request for showing tooltip for truncated text

**Comment by dobrinyonkov** [comment]
Closed with #8301

New tooltip property can be provided in the following way:
```html
<ui5-list>
    <ui5-li tooltip="Option #1">Option 1</ui5-li>
    <ui5-li tooltip="Option #2">Option 2</ui5-li>
    <ui5-li tooltip="Option #3">Option 3</ui5-li>
</ui5-list>
```
